src/modules/Blend/Module.js

```diff
@@ -0,0 +1,76 @@
+module.exports = function Dynamic(options, UI, util) {
+  
+  options = options || {};
+  options.func = options.func || "function(r1, g1, b1, a1, r2, g2, b2, a2) { return [ r1, g2, b2, a2 ] }";
+  
+  // Tell the UI that a step has been set up.
+  UI.onSetup(options.step);
+  var output;
+  
+  // This function is called on every draw.
+  function draw(input,callback,progressObj) {
+
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+
+    // Tell the UI that the step is being drawn
+    UI.onDraw(options.step);
+    var step = this;
+
+    // convert to runnable code:
+    if (typeof options.func === "string") eval('options.func = ' + options.func);
+
+    var getPixels = require('get-pixels');
+   
+    // save first image's pixels
+    var priorStep = util.getStep(-1);
```

Comment:
@tech4GT awesome -- next, can you use the `getStep()` API now available from #259 to fix this line, and remove the changes I'd attempted to implement `getStep()` in `AddStep.js`? Then let's do the big code tidying!

Reply:
@jywarren yeah I'll rebase these changes over that pr

Reply:
@jywarren Can I do that later today....actually I have a practical examination in 6 hours?